application/views/admin/emailtemplates/emailtemplates_view.php

```diff
@@ -23,14 +23,19 @@
     LSYii_ClientScript::POS_BEGIN);
 
 ?>
+<<<<<<< HEAD
+<div class="side-body <?php echo getSideBodyClass(false); ?>">
+    <h3 role="heading" aria-level="2"><?php eT("Edit email templates"); ?></h3>
+=======
```

Comment:
Hi, these <<<<<<< HEAD lines are unresolved conflicts.

Reply:
This is done!